packages/fetchai/skills/tac_control_contract/behaviours.py

```diff
@@ -56,6 +56,7 @@ def __init__(self, **kwargs):
         self._oef_msg_id = 0
         self._registered_desc = None  # type: Optional[Description]
         self.context.shared_state["is_items_created"] = False
```

Comment:
I am not quite sure why you use shared_state for intra skill state. It is intended for inter skill state. For intra skill you can just access the module, like `self.context.behaviours.behaviour_name`

Reply:
I find it easier just to use the shared_state. But if this is a bad practice I can change it.

Reply:
Yes I consider it bad practice as it gives other skills the ability to affect that skill.